build.gradle

```diff
@@ -114,7 +114,7 @@ subprojects { subproject ->
 	}
 
 	jacoco {
-		toolVersion = "0.7.2.201409121644"
+		toolVersion = "0.7.7.201606060606"
```

Comment:
Is this compatible with Sonar? I believe we tried to update this some time ago and the sonar builds failed.